applications/MeshingApplication/custom_utilities/meshing_utilities.cpp

```diff
@@ -21,6 +22,65 @@ namespace Kratos
 {
 namespace MeshingUtilities
 {
+void EnsureProperties(ModelPart& rModelPart)
```

Comment:
to me the name is not clear what it means
But I also don't have a better suggestion :)

Reply:
EnsureModelPartOwnsProperties?

Reply:
`EnsureModelPartHasProperties` would be more aligned with the other names of the functions in the Modelpart

Reply:
will you change this?